applications/RomApplication/tests/thermal_dynamic_test_files/PGROM/rom_data/RomParameters.json

```diff
@@ -2,6 +2,7 @@
     "rom_manager": false,
     "train_hrom": false,
     "run_hrom": false,
+    "assembling_strategy": "elemental",
     "rom_format":"numpy",
```

Comment:
Arent the results the same when using the elemental and the global PGRomB&S?

Reply:
Some tests have an elemental assembly strategy but a residual criterion. This is not compatible, as the residual is always 0 when building with an elemental approach. This means that the tests were created using a residual criterion, such that they misreach convergence in one iteration. If testing with global B&S, they will make more than one iteration and, therefore, yield different results. I have added this to the TODOs for updating the tests.